src/main/java/uk/gov/ida/stub/idp/StubIdpApplication.java

```diff
@@ -22,26 +22,13 @@
 import uk.gov.ida.saml.core.IdaSamlBootstrap;
 import uk.gov.ida.stub.idp.bundles.DatabaseMigrationBundle;
 import uk.gov.ida.stub.idp.configuration.StubIdpConfiguration;
-import uk.gov.ida.stub.idp.exceptions.mappers.CatchAllExceptionMapper;
-import uk.gov.ida.stub.idp.exceptions.mappers.FileNotFoundExceptionMapper;
-import uk.gov.ida.stub.idp.exceptions.mappers.IdpNotFoundExceptionMapper;
-import uk.gov.ida.stub.idp.exceptions.mappers.IdpUserNotFoundExceptionMapper;
+import uk.gov.ida.stub.idp.exceptions.mappers.*;
```

Comment:
Somewhere there exists a Java style doc for GDS that says that we don't use .* imports. It's a little out of date (see Sonar) but https://gds-way.cloudapps.digital/manuals/programming-languages/java.html#imports should give you an idea.